### DISCUSSION TITLE
Unable to retrieve auth token

### DISCUSSION THREAD

**Discussion by BrodyB** [blue-build/.github#31] [Support] [answered]
Hey, folks!

I'm just getting set up with a blue-build script to make modifications of Bluefin/Bazzite. I followed the instructions for building the recipes, creating a cosign signature and putting that secret into the repository settings, and I got a full and successful build.

However, when I try to `rpm-ostree rebase` with my new image, I get an error:
```
Pulling manifest: ostree-unverified-registry:ghcr.io/brodyb/ublue-brody-framework:latest
error: Creating importer: Failed to invoke skopeo proxy method OpenImage: remote error: unable to retrieve auth token: invalid username/password: unauthorized

~
```
As per the documentation, I didn't give a password to the cosign key. What could be causing this?

**Comment by BrodyB** [comment, marked as answer]
Hah, took me a while but I found [this issue](https://github.com/blue-build/website/issues/40) that explained the part I was missing!

**Reply by xynydev** [reply]
Glad that you found it. Cheers! :tada:
